spec/mailers/report_mailer_spec.rb

```diff
@@ -44,4 +45,31 @@
       end
     end
   end
+
+  describe "#scheduled_report" do
+    it "mails report on schedule", :elasticsearch do
+      deliveries.clear
+      owner = create(:user)
+      report = create(:report, query: { text: "something" }.to_json, user: owner)
+      scheduled_report = create(:scheduled_report, frequency: "daily", user: owner, report: report)
+
+      proposals = 3.times.map do |i|
+        tcr = create(:test_client_request, project_title: "something #{i}")
+        tcr.proposal.update(requester: owner)
+        tcr.proposal.reindex
```

Comment:
`reindex` pushes content to the ES server. 

`refresh` forces the ES server to re-open all its searcher objects to make recent updates accessible. https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-refresh.html